### DISCUSSION TITLE
Panda3DS 1 year anniversary edition (featuring Pokemon)

### DISCUSSION THREAD

**Discussion by wheremyfoodat** [wheremyfoodat/Panda3DS#277] [Announcements]
1 year since the first commit in the repo! This time I'm going to skip the formalities and just show some of what @noumidev and I have in store for you in this new update :p
(Thanks @UDXS for the ~~propaganda~~ photoshop)

![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/23dc325e-a6f8-43dd-aa70-cc07923e9562)

Happy pandiversary!

OK now the boring stuff
- Added SD card support
- Added CRO support
- Preliminary Amiibo work
- Initial CSND implementation (Gets various older homebrew such as minicraft working)
- Add SystemSaveData support (Gets OS apps like Mii Maker, Download Play, Activity Log, etc working)
- More FS work
- Fix graphical bugs in games that use invalid textures such as Pokemon X (feat @GPUCode)
- Various HLE work here and there
- More...

<hr /><em>This discussion was created from the release <a href='https://github.com/wheremyfoodat/Panda3DS/releases/tag/v0.6-pandiversary'>Panda3DS 1 year anniversary edition (featuring Pokemon)</a>.</em>

**Comment by wheremyfoodat** [comment]
![image](https://github.com/wheremyfoodat/Panda3DS/assets/44909372/6406b040-4264-4795-ae8f-7548a9ef25bd)

**Comment by liuk7071** [comment]
panda

**Comment by raphaelthegreat** [comment]
Omg :clap::clap:

**Comment by KostaSaizo7** [comment]
🎂
![image](https://github.com/wheremyfoodat/Panda3DS/assets/39852873/d8dc47ae-0bfa-438b-9961-1ff3088bee09)

**Comment by LaReunion974** [comment]
let's gooooooooooooooooooooooooooooooooooooooooooooooooo 👍 🍰

**Comment by LaReunion974** [comment]
![image](https://github.com/wheremyfoodat/Panda3DS/assets/102532446/9bc8ff9d-9d56-41db-9f2a-9996d97047e1)

**Comment by ghost** [comment]
It's the 1st anniversary in my timezone (Pacific Daylight)

**Comment by skylersaleh** [comment]
![](https://user-images.githubusercontent.com/102532446/268306435-9bc8ff9d-9d56-41db-9f2a-9996d97047e1.png)

**Comment by Wunkolo** [comment]
pemdas

**Comment by SleepingSnakezzz** [comment]
The Pokemon floodgates have opened. May panda have mercy upon thy souls.

**Reply by wheremyfoodat** [reply]
pemdas
